src/main/scala/com/imageintelligence/fs2sqs/AsyncSQS.scala

```diff
@@ -39,6 +39,18 @@ object AsyncSQS {
     }
   }
 
+  def changeMessageVisibilityAsync(client: AmazonSQSAsyncClient, request: ChangeMessageVisibilityRequest)(implicit s: Strategy): Task[ChangeMessageVisibilityRequest] = {
+    Task.async[ChangeMessageVisibilityRequest] { k =>
```

Comment:
Why is this A => Task[A]. You should return a `ChangeMessageVisibilityResponse`.